installers/win_installer.py

```diff
@@ -0,0 +1,359 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2020 Spyder Project Contributors.
+# Licensed under the terms of the MIT License
```

Comment:
Please change this to

    Licensed under the terms of the GPL-3.0 License

That's because the Mu project code is licensed GPL-3.0, so we can't change its license to MIT.